webhook_features/src/features/summary_comment.rs

```diff
@@ -188,6 +190,23 @@ async fn refresh_summary_comment(ctx: &Context, repo: Repository, pr_number: u64
 
     let cmt = util::get_metadata_sections_from_comments(&all_comments, pr_number);
 
+    let mut ignore_users: Vec<String> = vec![];
+    if let Some(cmt_id) = cmt.id {
```

Comment:
```suggestion
    let ignore_users: Vec<String> =
    if let Some(cmt_id) = cmt.id {
    ....} else { Vec::new() }
```

(nit), Can remove `mut`, not sure if it compiles, though

Reply:
Done